tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts

```diff
@@ -0,0 +1,684 @@
+      .withAlias(elementAlias)
+      .AsElementType()
+      .addGroup()
+        .withName("TestString")
+        .withAlias('testString')
+        .addTextBoxProperty()
+          .withLabel("Title")
+          .withAlias("title")
+        .done()
+      .done()
+      .build();
+    await umbracoApi.documentTypes.save(elementType);
+
+    return elementType;
+  }
+
+  async function createContentWithOneBlockListEditor(umbracoApi, documentName, documentAlias, elementName, elementAlias, blockListName, blockListAlias) {
+    const element = await createElement(umbracoApi, elementName);
+
+    const dataTypeBlockList = new BlockListDataTypeBuilder()
+      .withName(blockListName)
+      .addBlock()
+        .withContentElementTypeKey(element['key'])
+        .withSettingsElementTypeKey(element['key'])
+      .done()
+      .build();
+    const dataType = await umbracoApi.dataTypes.save(dataTypeBlockList);
+
+    const docType = new DocumentTypeBuilder()
+      .withName(documentName)
+      .withAlias(documentAlias)
+      .withAllowAsRoot(true)
+      .addGroup()
+        .withName('BlockListGroup')
+        .addCustomProperty(dataType['id'])
+          .withAlias(blockListAlias)
+        .done()
+      .done()
+      .build();
+    const generatedDocType = await umbracoApi.documentTypes.save(docType);
+
+    const rootContentNode = new ContentBuilder()
+      .withContentTypeAlias(generatedDocType["alias"])
+      .withAction("saveNew")
+      .addVariant()
+        .withName(blockListName)
+        .withSave(true)
+        .addProperty()
+          .withAlias(blockListAlias)
+          .addBlockListValue()
+            .addBlockListEntry()
+              .withContentTypeKey(element['key'])
+              .appendContentProperties(element.groups[0].properties[0].alias, "aliasTest")
+            .done()
+          .done()
+        .done()
+      .done()
+      .build()
+    await umbracoApi.content.save(rootContentNode);
+  }
+
+  test('can create content with a block list editor', async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+
+    const element = await createElement(umbracoApi, elementName);
+
+    const dataTypeBlockList = new BlockListDataTypeBuilder()
+      .withName(blockListName)
+      .addBlock()
+        .withContentElementTypeKey(element['key'])
+      .done()
+      .build();
+    const dataType = await umbracoApi.dataTypes.save(dataTypeBlockList);
+
+    const docType = new DocumentTypeBuilder()
+      .withName(documentName)
+      .withAlias(documentAlias)
+      .withAllowAsRoot(true)
+      .addGroup()
+        .withName('BlockListGroup')
+        .addCustomProperty(dataType['id'])
+          .withAlias(blockListAlias)
+          .withLabel(blockListName)
+        .done()
+      .done()
+      .build();
```

Comment:
Seems like this code and the code in `createContentWithOneBlockListEditor` is the same, we should probably have a local function called `createDocTypeWithBlockGridEditor` or something, so we do not duplicate code 👍